src/types/adapters.ts

```diff
@@ -1,7 +1,7 @@
 import { SupabaseClient } from "@supabase/supabase-js";
-import { Telegraf } from "telegraf";
+import { Telegram } from "telegraf";
 
 export type Adapters = {
   supabase: SupabaseClient;
-  telegram: Telegraf;
+  telegram: Telegram;
```

Comment:
```suggestion
  telegram: TelegramClient;
```

Why not maintain the same naming scheme? It seems the clearest.

Reply:
https://github.com/ubiquity/bounty-bot/pull/85/commits/c907fa63b1f770b9b23e329120dc9a7aaf626ca3